index.html

```diff
@@ -367,7 +367,7 @@ <h4>Community</h4>
             <li><em>Twitter:</em> <a href="https://twitter.com/openturns_org">@openturns_org</a></li>
             <li><em>StackOverflow:</em> <a href="https://stackoverflow.com/questions/tagged/openturns">[openturns]</a></li>
             <li><em>Publications:</em> <a href="https://arxiv.org/abs/1501.05242">baudin2015</a></li>
-            <li><em>Related software:</em> <a href="https://raven.inl.gov">RAVEN</a>, <a href="https://www.uqlab.com/">UQLab</a>, <a href="https://www.persalys.fr">Persalys</a></li>
```

Comment:
why did you remove these links ?

Reply:
Oups, I must fix this. Removing the link to Persalys was a mistake.

Reply:
I think they were here because their main goal is uncertainty quantification, not the case for dakota(optim) nor sparsegrid (doe only)

Reply:
The fact that Dakota is a UQ software is not my opinion : this is what is presented in the UQ community : https://www5.in.tum.de/wiki/index.php/SIAMUQ18_-_Slides_Minisymp_Software4UQ. SparseGrid++ is not only a DOE software. Please look at the doc https://sgpp.sparsegrids.org/ Almost all topics of UQ are covered by SparseGrid++. Moreover, Raven and UQLab are not technically related to OpenTURNS, whereas Persalys is. It would make more sense to indicate chaospy and Mystic (because they are in Python) or URANIE, because the CEA co-organizes the PRACE training with us. Isn'it?

Reply:
And there is Anatole3D : https://www.dps-fr.com/anatole (provided by @regislebrun )

Reply:
yes, that's what I said about dakota: its main goal is optim not uq
indeed sparsegrid does more that doe but it wouldnt say it covers all the fields of uq: distributions, running simulations etc
that's why I'd exclude it
I put these because they seemed relevant uq projects, I think it's a bit biased to include only python projects (I find it already deceiving that people using different ecosystems python/R/matlab dont talk to each other)
but imho yes chaospy or uranie are also relevant uq projects I'm totally ok to include them

Reply:
Ok. Fixed.

Reply:
Ok. Fixed.